content/en/blog/_posts/2022-08-25-pod-security/index.md

```diff
@@ -0,0 +1,90 @@
+```
+
+You get:
+
+```
+pods "admin-pod" is forbidden: violates PodSecurity "restricted:latest": unrestricted capabilities (container "admin" must not include "SYS_ADMIN" in securityContext.capabilities.add)
+```
+
+**Improved namespace warnings:**
+
+When modifying the `enforce` Pod Security labels on a namespace, PSA checks all existing pods for
+violations and surfaces warning if any are out of compliance. Now,
+[warnings are aggregated](https://github.com/kubernetes/kubernetes/pull/105889) for pods with
+identical violations, making large namespaces with many replicas much more manageable. For example:
+
+```
+Warning: frontend-h23gf2: allowPrivilegeEscalation != false
+Warning: myjob-g342hj (and 6 other pods): host namespaces, allowPrivilegeEscalation != false Warning: backend-j23h42 (and 1 other pod): non-default capabilities, unrestricted capabilities
+```
+
+Additionally, when applying a non-privileged to a namespace that has been
+[configured to be exempt](https://kubernetes.io/docs/concepts/security/pod-security-admission/#exemptions),
+you will now get a warning alerting you to this fact:
+
+```
+Warning: namespace 'kube-system' is exempt from Pod Security, and the policy (enforce=baseline:latest) will be ignored
+```
+
+**Changes to the Pod Security Standards**
+
+The [Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/),
+which are enforced by Pod Security admission, have been updated with support for the new Pod OS
+field. For restricted profiles in versions v1.25 and up, the following linux-specific fields will no
+longer be required when the pod's `.spec.os.name` has been explicitly set to `windows`:
+
+* Seccomp - The `seccompProfile.type` field on pod and/or container security contexts
+* Privilege escalation - The `allowPrivilegeEscalation` field on container security contexts
+* Capabilities - The requirement to drop `ALL` capabilities
+
+_Note that Kubelets prior to Kubernetes v1.24 did not enforce the Pod OS field, so clusters running
+older nodes should explicitly
+[pin restricted policies to a version](https://kubernetes.io/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces)
+prior to v1.25._
+
+
+## Migrating from Pod Security Policy to Pod Security Admission
+
+A [guide](https://kubernetes.io/docs/tasks/configure-pod-container/migrate-from-psp/) was published
+to make the process of migrating from PSP to PSA easier, and to help you choose the best migration
+strategy for your use case. In addition, a tool called
+[pspmigrator](https://github.com/kubernetes-sigs/pspmigrator) is being developed to automate parts
+of the migration process. We'll be talking about PSP migration in more detail at our upcoming
+Kubecon talk,
+[Migrating from Pod Security Policy](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/).
```

Comment:
```suggestion
[pspmigrator](https://github.com/kubernetes-sigs/pspmigrator) to automate parts
of the migration process. 

We'll be talking about PSP migration in more detail at our upcoming KubeCon 2022 NA talk,
*Migrating from Pod Security Policy*. Use the [KubeCon NA schedule](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/) to learn more.
```

Reply:
Done. Also added a direct link to the session.